docs/csharp/lambda-expressions.md

```diff
@@ -1,27 +1,148 @@
----
-title: Lambda Expressions
-description: Lambda expressions are shorthand syntax for methods and delegates. Learn when to use them.
-keywords: .NET, .NET Core
-author: dotnet-bot
-manager: wpickett
-ms.date: 06/20/2016
```

Comment:
@stevehoag Thanks. I completely forgot to copy the header from the original topic.